### ISSUE TITLE
Error installing in Pycharm

### ISSUE THREAD

**Issue by MahdiSayadSadr** [aidenlab/straw#101]
Collecting hic-straw
  Using cached https://files.pythonhosted.org/packages/a3/f1/19bb1d40f659e5daa98ba6a2d845c1203ec45661c1b855958643ccc37c66/hic-straw-1.1.3.tar.gz
Requirement already satisfied: pybind11>=2.4 in c:\user
(from hic-straw) (2.7.1)
Installing collected packages: hic-straw
  Running setup.py install for hic-straw: started
    Running setup.py install for hic-straw: finished with status 'error'

    ERROR: Command errored out with exit status 1:

    Complete output (12 lines):
    running install
    running build
    running build_ext
    building 'hicstraw' extension
    creating build
    creating build\temp.win32-3.8
    creating build\temp.win32-3.8\Release
    creating build\temp.win32-3.8\Release\src

    straw.cpp
    src/straw.cpp(34): fatal error C1083: Cannot open include file: 'curl/curl.h': No such file or directory
    error: command 'C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Professional\\VC\\Tools\\MSVC\\14.26.28801\\bin\\HostX86\\x86\\cl.exe' failed with exit status 2
    ----------------------------------------
ERROR: Command errored out with exit status 1: externally-managed --compile Check the logs for full command output.
WARNING: You are using pip version 19.2.3, however version 22.0.3 is available.
You should consider upgrading via the 'python -m pip install --upgrade pip' command.

**Comment by chorzow** [comment]
@MahdiSayadSadr you need to have curl.h library installed (see [this](https://groups.google.com/g/3d-genomics/c/i1Ukhwf1-mY/m/M1dQCIsgAgAJ) answer in Google groups).

TL;DR: executing `sudo apt-get install libcurl4-gnutls-dev` with following installation from Pycharm worked for me.

**Comment by fage66639** [comment]
"I have the same question. Has it been resolved?"

**Comment by fage66639** [comment]
win11 python3.9
